tools/lib/routes-loader.js

```diff
@@ -16,6 +18,79 @@ export default function(source) {
     source = source.replace('import \'babel/polyfill\';', ''); // eslint-disable-line no-param-reassign
   }
 
+  const routeGenerator = new Promise((resolve, reject) => {
+    glob('**/*.json', { cwd: join(__dirname, '../../routes') }, (err, files) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(files);
+      }
+    });
+  })
+  .then((files) => {
+    if (files.length === 0) {
+      return Promise.resolve([]);
+    }
+
+    return Promise.all(
+      map(files, perFile => new Promise((resolve, reject) => {
+        const file = join(__dirname, '../../routes', perFile);
+        fs.readFile(file, 'utf8', (err, data) => {
+          if (err) {
+            reject(err);
+            return;
+          }
+
+          try {
+            resolve(JSON.parse(data));
+          } catch(e) {
+            reject(e);
+          }
+
+        });
+      }))
+    );
+  })
+  .then((routeInfo) => {
+    let factories = {};
+    let lines = [];
+    forEach(routeInfo, ({factory, routes}) => {
+      factories[factory.name] = factory.path;
+
+      forEach(routes, (perRouteData, perRouteName) => {
+        const contentParam = `content: require("${perRouteData['content_path']}"),`;
+        let menuParam = '';
+        if (perRouteData['menu_path']) {
+          menuParam = `menu: require("${perRouteData['menu_path']}"),`;
+        }
+
+        let line = '';
+        if (target === 'node') {
+          line += `'${perRouteName}': () => ${factory.name}({`;
+          line += `  name: "${perRouteData.name}",`;
+          line += `  title: "${perRouteData.title}",`;
+          line += `  ${contentParam}`;
+          line += `  ${menuParam}`;
+          line += `}),`;
+        } else {
+          line += `'${perRouteName}': () => Promise.resolve(`;
+          line += `  ${factory.name}({`;
+          line += `    name: "${perRouteData.name}",`;
+          line += `    title: "${perRouteData.title}",`;
+          line += `    ${contentParam}`;
+          line += `    ${menuParam}`;
+          line += `  })`;
+          line += `),`;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

You can save the `line +=`